### PR TITLE
Fix install script.sh

### PR DESCRIPTION
**Pull request by vidarl**
Fixes:
 - Removed superfluous ; and indenting ifs in install_script.sh
 - Fixed so that install_script.sh exits properly if composer install fails

### CODE REVIEW

**Comment by andrerom** [comment]
But is Admin UI with Redis and Varnish supposed to fail?

**Comment by lserwatka** [comment]
They should not, Vidar you need to check it.

**Comment by vidarl** [comment]
Travis issues fixed.

I rebased branch too, but there was a small problem in https://github.com/ezsystems/ezplatform/pull/464/files#diff-e93766272f057b168d428064ad212264R9. 

```diff
-if [ "$s" != "" ]; then
+if [ "$s" != "0" ]; then
```

**Comment by andrerom** [comment]
@lserwatka / @mnocon Ready for merge? If so anything else that should be merged so we reduce amount of repo merges :P

**Comment by mnocon** [comment]
Looks good to me! (and I'm not aware of anything else to merge to metarepo)

File: doc/docker/install_script.sh [hunk 1]
```diff
@@ -3,19 +3,27 @@
...
+    echo 'Dumping database into doc/docker/entrypoint/mysql/2_dump.sql for use by mysql on startup.'
+    mysqldump -u $DATABASE_USER --password=$DATABASE_PASSWORD -h $DATABASE_HOST --add-drop-table --extended-insert  --protocol=tcp $DATABASE_NAME > doc/docker/entrypoint/mysql/2_dump.sql
 fi;
 
-rm -Rf var/logs/* var/cache/*/*;
-chown -R www-data:www-data var/cache var/logs web/var;
-find var/cache var/logs web/var -type d -print0 | xargs -0 chmod -R 775;
-find var/cache var/logs web/var -type f -print0 | xargs -0 chmod -R 664;
-chown -R www-data:www-data app/config src;
+rm -Rf var/logs/* var/cache/*/*
```
Comment (wizhippo) [inline doc/docker/install_script.sh]: Should be `log`  not `logs`

**Comment by lserwatka** [comment]
@vidarl @andrerom have you see comment from @wizhippo ? 

+rm -Rf var/logs/ var/cache/*/
Should be log not logs

**Comment by andrerom** [comment]
Which comment? Are you referring to his PR against master?

**Comment by lserwatka** [comment]
I had email notification, and link is pointing to this PR. He commented at 15:16 today. Dunno which branch was it.

**Comment by vidarl** [comment]
I already have a pending PR for the "logs" vs "log" issue ( + other fixes ):
https://github.com/ezsystems/ezplatform/pull/455

Now that this PR has been merged, I'll rebase 455

**Pull request merged**